vividus-bdd-engine/src/main/java/org/vividus/bdd/order/StoryOrder.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.order;
+
+import static org.apache.commons.lang3.Validate.isTrue;
+
+import java.util.Comparator;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+import org.jbehave.core.model.Story;
+
+/**
+ * Methods of this class are expected to be used in test projects as static factory methods for story comparators
+ */
+public final class StoryOrder
```

Comment:
`MetaBasedStoryExecutionPriority`

Reply:
renamed, but here is a risk that we will want to add some more ordering strategies and will have to rename this class to something more generic leading to difficulties in support